cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpCache.java

```diff
@@ -0,0 +1,36 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import okhttp3.Cache;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+final class CwmsHttpCache {
+    static final String CACHE_SIZE_PROPERTY_KEY = "cwms.http.client.cache.size";
+    static final String CACHE_DIRECTORY_PROPERTY_KEY = "cwms.http.client.cache.directory";
+    static final String CACHE_DEFAULT_DIRECTORY = System.getProperty("user.home") + "/AppData/Local/CwmsHttpClient/Cache";
```

Comment:
This directory is specific to Windows OS, I'd go with something like `Paths.get(System.getProperty("java.io.tmpdir").resolve("CWMS").resolve("cwms-http-client").resolve("cache")`